schedule/migration/aarch64_regression_test_offline.yaml

```diff
@@ -46,20 +44,23 @@ conditional_schedule:
         - console/system_prepare
         - console/check_os_release
         - console/check_system_info
-        - console/check_migration_features
+        - '{{check_migration_features}}'
         - console/check_network
         - console/system_state
         - console/prepare_test_data
         - console/consoletest_setup
         - '{{check_upgraded_service}}'
-        - '{{openldap_to_389ds}}'
```

Comment:
From what I understood from the progress ticket, this should not be removed:  `modules like migration/openldap_to_389ds or console/orphaned_packages_check or basic checks of the migrated system should remain there`

Reply:
Also applies for the other yamls where `openldap_to_389ds` is removed.

Reply:
openldap_to_389ds is only tested on x86_64, for the other arches, it is not supported and I checked the cases' setting, no case set OPENLDAP_TO_389DS=1. So I remove it from the other three arches' yamls directly.